Users/Glob_Patterns.rst

```diff
@@ -123,14 +121,10 @@ brackets. Parentheses that have no match are ignored as well as
     False
```

Comment:
We could delete this example too. I don't find the purpose of it. This is my opinion. Don't do it until someone else says that it's ok.